src/components/ui/TripCard/AccordionDetails/DetailsGrid/DetailItem/DetailItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { Stack, Typography } from '@mui/material';
+import { DetailsItemWrapper } from './DetailItem.styled';
+
+interface IDetailsItemProps {
+    title: string;
+    value: string;
+    imageSrc: string;
+    imageAlt: string;
+}
+export const DetailsItem = ({
+    title,
+    value,
+    imageSrc,
+    imageAlt,
+}: IDetailsItemProps) => {
+    return (
+        <DetailsItemWrapper direction={'row'} spacing={3}>
+            <Stack direction={'row'} spacing={1}>
+                <img src={imageSrc} alt={imageAlt} />
```

Comment:
Do we have any scenario where the image src URL will get as empty? what will be shown in the UI at the time?

Reply:
The image source URL doesn't become empty, as the elements are fixed and won't change. These are part of the layout.